README.md

```
@@ -1,2 +1,70 @@
+```
+
+## Usage
+
+1. Start the server using `make run`
+2. Navigate to http://localhost:8080/index.html in your web browser
+   - Replace `index.html` with any file in the server directory you want to access
+
+## How It Works
+
+The server:
+1. Creates a socket and binds it to port 8080
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)